### ISSUE TITLE
Implement CI/CD for k8s 

### ISSUE THREAD

**Issue by haosanzi** [alibaba/inclavare-containers#364]
Need to implement integration testing on k8s environment(centos and ubuntu).

The workflow will be triggered nightly.

The steps of workflow include:
- build k8s running environment
- create pods for various images
- test pod function
- clear k8s environment

**Comment by haosanzi** [comment]
- Right now k8s on centos support to run and test hello-world and golang-web pod. Need to add other test cases.
- In addition, the `clear k8s environment` in `k8s_centos.yml` lacks a clear network environment process.
For example, `ifconfig | grep veth` in the host will input many veth.
```c
vethf616d106: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1450
vethf66056ed: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1450
vethf718fe59: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1450
vethfb64246a: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1450
```

**Comment by liying-000** [comment]
Deadline: the end of this month.

**Comment by haosanzi** [comment]
Solved in #464
